### ISSUE TITLE
Missing config option to disable hungarian-notation and trailing-whitespace

### ISSUE THREAD

**Issue by m2f2liberoit** [barklan/inline_sql_syntax#122]
Great extension, but I'm pestered with fake errors -- I simply cannot find where to set "hungarian-notation", and "trailing-whitespace" ignore errors settings for sql-lint.

I tried with .sql-lint.json and ~/.config/sql-lint/config.json = ~ = c:\users\myself - no luck
How? where?

Thanks!

Windows 11
VSCODE Version: 1.94.2 (user setup)
Commit: 384ff7382de624fb94dbaf6da11977bba1ecd427
Date: 2024-10-09T16:08:44.566Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by arpanpreneur** [comment]
Yeah these errors are really annoying. I have tried the .sql-lint.json file as suggested in sql-lint's respository, but it seems it doesn't respect that. This should be as simple as making a new settings config option and passing it down to the sql-lint library.
